packages/docgen/src/test/fixtures/markdown/code.js

```diff
@@ -1,3 +1,16 @@
+let _lastSum;
```

Comment:
For what it's worth, these test fixtures aren't super valuable in their current form, since this could be totally wrong, and the tests still wouldn't fail, because the output isn't regenerated as part of the test, but presumably as a separate manual process.

Related: https://github.com/WordPress/gutenberg/pull/13329#discussion_r278552883

Reply:
I don't think this particular fixture is used anywhere. For Markdown we should add the test [here](https://github.com/WordPress/gutenberg/blob/master/packages/docgen/src/test/formatter-markdown.js).

Reply:
> I don't think this particular fixture is used anywhere. For Markdown we should add the test [here](https://github.com/WordPress/gutenberg/blob/master/packages/docgen/src/test/formatter-markdown.js).

But it's not the formatter which is responsible to exclude the private DocBlocks, it occurs in the `engine.js` filtering.

Reply:
> I don't think this particular fixture is used anywhere.

Should it be removed here then?

Reply:
> Should it be removed here then?

Yes.

> But it's not the formatter which is responsible to exclude the private DocBlocks, it occurs in the engine.js filtering.

Oh, I see now, commented too quickly. The ignored symbols are actually filtered out by the CLI, not the formatter or the engine, so it's more tricky to test.

Filtering symbols (by name or tag) shouldn't be something the CLI takes care of. Ideally, we would push this code down to `get-intermediate-representation.js`. For the purpose of shipping this, I'm fine if we defer that refactor until later provided that we create an issue to track it.